src/components/Explore/LogsByService/Tabs/FieldsBreakdownScene.tsx

```diff
@@ -369,7 +369,7 @@ function buildNormalLayout(variable: CustomVariable) {
               body: new SceneReactObject({
                 reactNode: <LoadingPlaceholder text="Loading..." />,
               }),
-            })            
```

Comment:
I think webstorm has a bug where you cannot turn off autoformatting on save. Very frustrating